tsl/test/isolation/expected/compression_ddl.out

```diff
@@ -372,25 +372,28 @@ lock_chunktable
 (1 row)
 
 step RC1: 
-  BEGIN;
-  SELECT
-    recompress_chunk(ch.schema_name || '.' || ch.table_name)
-  FROM _timescaledb_catalog.hypertable ht, _timescaledb_catalog.chunk ch
-  WHERE ch.hypertable_id = ht.id AND ht.table_name like 'ts_device_table'
-  ORDER BY ch.id LIMIT 1;
-
+  DO $$
+  DECLARE
+    chunk_name text;
+  BEGIN
+  FOR chunk_name IN
+      SELECT ch.schema_name || '.' || ch.table_name
+	FROM _timescaledb_catalog.hypertable ht,
+	     _timescaledb_catalog.chunk ch
+       WHERE ch.hypertable_id = ht.id
+	 AND ht.table_name like 'ts_device_table'
```

Comment:
Where does this `AND` align? I'm confused with the indentation here.

Reply:
There is a tab in the specification file, causing this strange indentation. Removed all tabs in the file.